share/templates/categories.json

```diff
@@ -0,0 +1,93 @@
+{
+  "categories": [
```

Comment:
I believe either does not need this tag (just `[]` as the root object), or the file should be called `templates.json`

Reply:
removed this tag